Esri/arcgis-maps-sdk-swift-samples
# [New] Edit feature attachments

## Description

This PR implements **`Edit feature attachments`** in `Edit and Manage Data` category.

`swift/issues/5565`

## How To Test
Load map and wait for features to be placed. Tap on feature to show callout. To add an attachment press the callout button and load the attachment sheet. The attachments should sync with the server. 

## Screenshots
![edit-feature-attachments](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/13937686/a9f3bb7a-63cc-4630-b953-f4a94ccb71af)
